Lib/glyphsLib/interpolation.py

```diff
@@ -68,9 +69,13 @@ def build_designspace(masters, master_dir, out_dir, instance_data):
     tmp_path = os.path.join(master_dir, 'tmp.designspace')
     writer = DesignSpaceDocumentWriter(tmp_path)
 
-    base_family, base_style = add_masters_to_writer(writer, masters)
+    instances = filter(is_instance_active, instance_data.get('data', []))
```

Comment:
oh, it's because `filter` returns an iterator in python 3, not a list, but you are reusing `instances` later in `add_instances_to_writer`; however by that time the iterator has already been consumed by `get_axes` and it's empty. You should probably do `list(filter(...))` so you can iterate more than once